Multihop workers doc (#2772)
hashicorp/boundary: website/content/docs/concepts/filtering/worker-tags.mdx

# Target Worker Filtering

> Once workers have tags, these tags may be used to control which

Reviewer: ```suggestion
Once workers have tags, you can use these tags to control which
```

workers are allowed to manage a given session by specifying optional worker filter attributes

> when configuring [targets](/docs/concepts/domain-model/targets).

Reviewer: ```suggestion
when you configure [targets](/docs/concepts/domain-model/targets).
```

The `egress_worker_filter` attribute controls which workers will be used for egress to a target. This is the worker

> that will access the target.

Reviewer: ```suggestion
that accesses the target.
```

The `ingress_worker_filter`<sup>HCP Only</sup> attribute controls which workers will be used for ingress to a target.

> This is the worker a client will connect to when initiating a connection.

Reviewer: ```suggestion
This is the worker a client connects to when initiating a connection to a target.
```
